backend/LeetWars.Core/LeetWars.Core.Common/DTO/User/UserLoginDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace LeetWars.Core.Common.DTO.User
+{
+    public class UserLoginDto
+    {
```

Comment:
where do we use this class?

Reply:
I used only in frontend part for now, so I thought to add it in backend

Reply:
I believe we don't need it, as we do not send password to our backend